bvm/ballerina-core/src/main/java/org/ballerinalang/model/values/BMap.java

```diff
@@ -327,6 +330,32 @@ public BType getType() {
         return this.type;
     }
 
+    @Override
+    public void stamp(BType type) {
+
```

Comment:
Shall we remove this extra line added after the function signature?

Reply:
Fixed.